src/components/BackgroundParallex.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useRef, useEffect } from "react";
```

Comment:
You can omit React and the line will be `import { useRef, useEffect } from 'react';` Because since React version 18, you don't need to explicity import React anymore (that's the version we're using)